ansible/roles/distributed_press/templates/nginx-static.j2

```diff
@@ -12,26 +12,36 @@ server {
 
   error_page 404 /404.html;
 
+  add_header Cache-Control no-cache;
   add_header Strict-Transport-Security "max-age=31536000; includeSubDomains; preload" always;
-  add_header 'X-Frame-Options' 'ALLOW-FROM *';
+  add_header X-Frame-Options "sameorigin";
```

Comment:
I think we want to allow iframes for sites by default

Reply:
this only forbids the site to be iframed elsewhere (like a malicious site hijacking clicks)